src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.head_a {
+  height: 60px;
+}
+
+.header img {
+  margin: 10px 0;
+}
```

Comment:
You are styling an element directly using a tag name (img). According to the checklist, you should avoid this and use classes instead.